python/hail/methods/impex.py

```diff
@@ -598,23 +617,72 @@ def import_table(paths, key=[], min_partitions=None, impute=False, no_header=Fal
 def import_plink(bed, bim, fam, min_partitions=None, delimiter='\\\\s+',
                  missing='NA', quant_pheno=False, a2_reference=True, reference_genome=None,
                  contig_recoding={'23': 'X', '24': 'Y', '25': 'X', '26': 'MT'}, drop_chr0=False):
-    return Env.hc().import_plink(bed, bim, fam, min_partitions, delimiter,
-                                 missing, quant_pheno, a2_reference, reference_genome, contig_recoding,
-                                 drop_chr0).to_hail2()
+    rg = reference_genome if reference_genome else Env.hc().default_reference
+
+    if contig_recoding:
+        contig_recoding = TDict(TString(), TString())._convert_to_j(contig_recoding)
+
+    jmt = Env.hc()._jhc.importPlink(bed, bim, fam, joption(min_partitions), delimiter,
+                                    missing, quant_pheno, a2_reference, rg._jrep,
+                                    joption(contig_recoding), drop_chr0)
+
+    return MatrixTable(jmt)
 
 
 @handle_py4j
 @typecheck(path=oneof(strlike, listof(strlike)),
            drop_cols=bool,
            drop_rows=bool)
 def read_matrix(path, drop_cols=False, drop_rows=False):
-    return Env.hc().read(path, drop_cols, drop_rows).to_hail2()
+    return MatrixTable(Env.hc()._jhc.read(path, drop_cols, drop_rows))
 
 
 @handle_py4j
 @typecheck(path=oneof(strlike, listof(strlike)))
 def get_vcf_metadata(path):
-    return Env.hc().get_vcf_metadata(path)
+    """Extract metadata from VCF header.
+
+    Examples
+    --------
+
+    >>> metadata = hc.get_vcf_metadata('data/example2.vcf.bgz')
+
+    Notes
+    -----
+
+    This method parses the VCF header to extract the `ID`, `Number`,
+    `Type`, and `Description` fields from FORMAT and INFO lines as
+    well as `ID` and `Description` for FILTER lines. For example,
+    given the following header lines:
+
+    .. code-block:: text
+
+        ##FORMAT=<ID=DP,Number=1,Type=Integer,Description="Read Depth">
+        ##FILTER=<ID=LowQual,Description="Low quality">
+        ##INFO=<ID=MQ,Number=1,Type=Float,Description="RMS Mapping Quality">
+
+    The resulting Python dictionary returned would be
+
+    .. code-block:: python
+
+        metadata = {'format': {'DP': {'Number': '1', 'Type': 'Integer', 'Description': 'Read Depth'}},
+                    'filter': {'LowQual': {'Description': 'Low quality'}},
+                    'info': {'MQ': {'Number': '1', 'Type': 'Float', 'Description': 'RMS Mapping Quality'}}}
+
+    which can be used with :meth:`.export_vcf` to fill in the relevant fields in the header.
+    
+    Parameters
+    ----------
+    path : str or list of str
```

Comment:
kinda like using the 
```
:obj:`str` or :obj:`list` of :obj:`str`
```
for how it renders. Maybe it's too hard to read as plain text? I dunno

Reply:
why does this read multiple files? That seems like the wrong design. It should just take one. Can we fix that here, or should we do in a separate PR?

Reply:
It just parses the first one.  Will fix.

Reply:
OK, fixed.